aroma/features.py

```diff
@@ -73,6 +74,8 @@ def feature_time_series(mel_mix, mc):
     for i in range(nsplits):
         # Select a random subset of 90% of the dataset rows
         # (*without* replacement)
+        if "pytest" in sys.modules:  # detects we are using pytest
+            np.random.seed(i)
```

Comment:
I don't love the idea of having test-specific code in the main package, although I can see why you need it here. What if we just set the seed based on the iteration here for _all_ cases?

Reply:
I have moved the seed to the test file. I think it's cleaner this way. WDYT?

Reply:
Agreed!